src/busco/config.vsh.yaml

```
@@ -0,0 +1,210 @@
+functionality:
+  name: busco
+  description: Assessment of genome assembly and annotation completeness with single copy orthologs
+  info:
+    keywords: [Genome assembly, quality control]
+    homepage: https://busco.ezlab.org/
+    documentation: https://busco.ezlab.org/busco_userguide.html
+    repository: https://gitlab.com/ezlab/busco
+    reference: "10.1007/978-1-4939-9173-0_14"
+    licence: MIT
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: --input
+          alternatives: ["-i"]
+          type: file
+          description: |
+            Input sequence file in FASTA format. Can be an assembled genome or transcriptome (DNA), or protein sequences from an annotated gene set. Also possible to use a path to a directory containing multiple input files.
+          required: true
+          example: file.fasta
+        - name: --mode
+          alternatives: ["-m"]
+          type: string
+          choices: [genome, geno, transcriptome, tran, proteins, prot]
+          required: true
+          description: |
+            Specify which BUSCO analysis mode to run. There are three valid modes:
+
+              - geno or genome, for genome assemblies (DNA)
+              - tran or transcriptome, for transcriptome assemblies (DNA)
+              - prot or protein, for annotated gene sets (protein)
+          example: protein
```

It seems BUSCO has a [list of synonyms](https://gitlab.com/ezlab/busco/-/blob/master/src/busco/BuscoConfig.py?ref_type=heads#L719-750) that they use to map one to the other, but `protein` seems to be an alias of `proteins` and not the other way around.

```suggestion
        - name: --mode
          alternatives: ["-m"]
          type: string
          choices: [genome, geno, transcriptome, tran, proteins, prot]
          required: true
          description: |
            Specify which BUSCO analysis mode to run. There are three valid modes:
              - geno or genome, for genome assemblies (DNA)
              - tran or transcriptome, for transcriptome assemblies (DNA)
              - prot or proteins, for annotated gene sets (protein)
          example: proteins